shared/src/lib.rs

```diff
@@ -49,7 +49,7 @@ pub const ENCODED_VOUCHER_LEN: usize = 1 + MAC_LENGTH; // 1 byte for the length
 // maximum supported length of connection identifier for R
 pub const MAX_KDF_CONTEXT_LEN: usize = 150;
 pub const MAX_KDF_LABEL_LEN: usize = 15; // for "KEYSTREAM_2"
-pub const MAX_BUFFER_LEN: usize = 256;
+pub const MAX_BUFFER_LEN: usize = 256 + 32;
```

Comment:
Is this intentional or just an artifact of debugging with credentials that happen to be a bit longer than needed? (A CRED_x that does not carry the full dash separated MAC address may be short enough)

Reply:
Solved